### PR TITLE
Helpr

### LINKED ISSUE #63: List of current commands that need refactoring
Functionality
1. `cct.js` -->Honestly could remove this role since it is not really important

UI
1. `createStudyChannel.js`
1. `deleteStudyChannel.js`
1. `joinStudyChannel.js`
1. `help.js`

### PR DESCRIPTION
**Pull request by RTX-Banana**
Refactored s!help and part of s!git commands.
Reordered the parameters for all commands to match the order shown in Command.js.
Added disabled parameter and implemented disabled functionality in CommandHandler.js.
Resolves #63

### CODE REVIEW

**Comment by jerrylee17** [comment]
Converted to draft - still work in progress

File: src/commands/util/help.js [hunk 1]
```diff
@@ -71,17 +71,36 @@ module.exports = new Command({
...
+          helpEmbed.setTitle(capitalize(infoText));
+          return;
+        }
+
+        if (field == 'aliases') {
+          infoText = info.join(', ');
+          noAlias = false;
+        }
         if (field != 'name' && field != 'aliases' && field != 'example') {
           infoText = capitalize(infoText);
```
Comment (charlotte-zhuang) [inline src/commands/util/help.js:90]: Crashes if `field === 'disabled'`. The `disabled` field has a boolean value.
For example, try `s!help github`

Comment (charlotte-zhuang) [reply src/commands/util/help.js:90]: Maybe it would be best to add a type check in the `capitalize` function.

**Review by charlotte-zhuang — CHANGES_REQUESTED** [review]
Disabled commands are causing some issues in `src/commands/util/help.js`.
1. Line 90 crashes if `typeof infoText !== String`. I think this can be solved with a type check in the `capitalize` function on line 16.
2. Disabled commands are being displayed in `s!help`. On line 43, a check for `tmp[j].disabled` should do the trick.

**Review by jerrylee17 — APPROVED** [review]
Tested and it looks good to me!

**Review by charlotte-zhuang — APPROVED** [review]
Nice!

**Pull request merged**